src/platform/forms/helpers.js

```diff
@@ -38,14 +38,16 @@ export function groupPagesIntoChapters(routes, prefix = '') {
   }));
 }
 
-export function isInProgress(pathName) {
+export function isInProgress(pathName, additionalSafePaths = []) {
```

Comment:
I refactored this but added the unit tests first, based on the old function, to confirm I didn't break stuff.